docs/src/services/xds.md

```diff
@@ -24,36 +24,35 @@ In addition to static configuration provided upon startup, a Quiklin proxy's con
 
 - **Cluster Discovery Service [(CDS)][CDS]**: Provides information about known clusters and their membership information.
-  * The proxy uses these resources to discover clusters and their endpoints.
-  * While cluster topology information like [locality] can be provided in the configuration, the proxy currently does not use this information (support may be included in the future however).
-  * Any [load balancing information][lbpolicy] included in this resource is ignored. For load balancing, use [Quilkin filters][filters-doc] instead.
-  * Only [cluster discovery type] `STATIC` and `EDS` is supported. Configuration including other discovery types e.g `LOGICAL_DNS` is rejected.
+  - The proxy uses these resources to discover clusters and their endpoints.
+  - While cluster topology information like [locality] can be provided in the configuration, the proxy currently does not use this information (support may be included in the future however).
+  - Any [load balancing information][lbpolicy] included in this resource is ignored. For load balancing, use [Quilkin filters][filters-doc] instead.
+  - Only [cluster discovery type] `STATIC` and `EDS` is supported. Configuration including other discovery types e.g `LOGICAL_DNS` is rejected.
 
 - **Endpoint Discovery Service [(EDS)][EDS]**: Provides information about endpoints.
-  * The proxy uses these resources to discover information about endpoints like their IP addresses.
-  * Endpoints may provide [Endpoint Metadata][endpoint-metadata] via the [metadata][xds-endpoint-metadata] field. These metadata will be visible to filters as part of the corresponding endpoints information when processing packets.
-  * Only [socket addresses] are supported on an endpoint's address configuration - i.e an IP address and port number combination. Configuration including any other type of addressing e.g named pipes will be rejected.
-  * Any [load balancing information][clapolicy] included in this resource is ignored. For load balancing, use [Quilkin filters][filters-doc] instead.
+  - The proxy uses these resources to discover information about endpoints like their IP addresses.
+  - Endpoints may provide [Endpoint Metadata][endpoint-metadata] via the [metadata][xds-endpoint-metadata] field. These metadata will be visible to filters as part of the corresponding endpoints information when processing packets.
+  - Only [socket addresses] are supported on an endpoint's address configuration - i.e an IP address and port number combination. Configuration including any other type of addressing e.g named pipes will be rejected.
+  - Any [load balancing information][clapolicy] included in this resource is ignored. For load balancing, use [Quilkin filters][filters-doc] instead.
```

Comment:
I don't think we use these types from envoy anymore, I think it would be good to include some info and a link to the types protobuf instead.

Reply:
The only other thing I found that might be worth playing with was https://github.com/matze/protoc-gen-mdbook - to generate docs from the proto files themselves.

Just an idea.